docs/api/rpc-params.md

```diff
@@ -0,0 +1,158 @@
+
+- `account_id`: (string) NEAR [account ID format](/docs/concepts/account)
+- `prefix`: (base64 string) optional string used to segment our key-value store
+- `public_key` (string) the public part of your public/private key pair
+- `method_name`: (string) the name of the RPC method you're calling
+
+## Common Parameters
+
+- `block_id`: (integer) block height or block hash
+- `finality`: (string) filter for status of block finalization [ `optmistic` | `near-final` | `final` ] 
+
+
+*At time of writing, `finality` is only available on StageNet.*
+
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+
+### Using block height
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using block hash
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `optimistic` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `near-final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "near-final"
+             }'
+```
+
+### Using `final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "final"
+             }'
+```
+
+<blockquote class="info">
+<strong>did you know?</strong><br><br>
+
+The `request_type` parameter can take any one of the following values (and associated parameters)
+
+```rust
+view_account          account_id: string
+view_state            account_id: string, prefix: string /* base64 */
+view_access_key       account_id: string, public_key: string
+view_access_key_list  account_id: string
+call_function         account_id: string, method_name: string, args: string /* base64 */
+```
```

Comment:
It is hard to find the right way to document this, but I think this block can be hard to read since it is not immediately clear that this is a table. Maybe a nested list or a list of tables will serve us better.

Here is an example of a table I have in mind:

![image](https://user-images.githubusercontent.com/304265/76376344-d748c900-6326-11ea-9b04-d378ec326729.png)

Here is the nested list: https://github.com/nearprotocol/nearcore/pull/2148 (under API section)

Neither of the options seems ideal given that we have 1-3 arguments here. Maybe, we should just be sparser and go with a full example of all the parameters for each of the available types.

Reply:
✅